db/migrate/20210307064615_create_subjects.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSubjects < ActiveRecord::Migration[6.1]
+  def change
+    create_table :subjects do |t|
+      t.string :name
```

Comment:
i could have added `null: false` here, but i think the model validations takes care of that. i'm not sure what is standard practice.